stripe-crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/model/OnrampConfigurationCallback.kt

```diff
@@ -0,0 +1,29 @@
+package com.stripe.android.crypto.onramp.model
+
+/**
+ * Callback invoked after attempting to configure the Onramp flow.
+ */
+fun interface OnrampConfigurationCallback {
```

Comment:
Would it be more ergonomic to nest these in the class they're used instead?

Reply:
I'm not sure. From a namespace perspective yes, but from legibility perspective I think no. I believe there's intended to be quite a few more callbacks, so I'd worry that putting them all together along with the Coordinator would make it difficult to parse. Happy to move it if that's preferred though.